examples/NDSL/02_NDSL_basics.ipynb

```diff
@@ -25,16 +25,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from basic_boilerplate import get_one_tile_factory, plot_field_at_k0\n",
     "from ndsl import StencilFactory\n",
+    "from ndsl.boilerplate import get_factories_single_tile_numpy, plot_field_at_kN\n",
     "\n",
     "nx = 6\n",
     "ny = 6\n",
     "nz = 1\n",
     "nhalo = 1\n",
-    "backend=\"numpy\"\n",
     "\n",
-    "stencil_factory: StencilFactory = get_one_tile_factory(nx, ny, nz, nhalo, backend)"
+    "stencil_factory, _ = get_factories_single_tile_numpy(nx, ny, nz, nhalo)"
```

Comment:
Unsure about python syntax here (about the `_`): is this common in python or is there another convention for unused destructured arguments?

Reply:
I think that still the convention, or at least I am unaware of something better